client/virtual_channel.go

```diff
@@ -156,6 +201,35 @@ func (c *Client) persistVirtualChannel(ctx context.Context, parent *Channel, pee
 	return ch, nil
 }
 
+func (c *Channel) pushVirtualUpdate(ctx context.Context, state *channel.State, sigs []wallet.Sig) (err error) {
+	if !c.machMtx.TryLockCtx(ctx) {
+		err = errors.Errorf("locking machine mutex in time: %v", ctx.Err())
+		return
+	}
+	defer c.machMtx.Unlock()
+
+	m := c.machine
+	err = m.ForceUpdate(ctx, state, hubIndex)
```

Comment:
This is just a convention. The hub is not a participant, it does not really have a participant index. But to make this work with our persistence logic, we need to set a participant index. Could be the basis for another issue. Our persistence logic could need some revision.

Reply:
Yea that sounds a bit sketchy.  
I think the persistence was designed before we did SubC/VC so it may not be conceptually designed for it.  
Should i open an issue or is it still manageable?

Reply:
You can open an issue. `hubIndex` was introduced before, so it should not be in the focus of this review.

Reply:
Done in #138.